web/src/engine/websites/Tenkai.ts

```diff
@@ -0,0 +1,36 @@
+import { Tags } from '../Tags';
+import icon from './Tenkai.webp';
+import { DecoratableMangaScraper } from '../providers/MangaPlugin';
+import * as Common from './decorators/Common';
+
+function MangaLabelExtractor(element: HTMLMetaElement) {
+    return element.content.split('|').at(0).trim();
+}
+function MangaInfoExtractor(anchor: HTMLAnchorElement) {
+    return {
+        id: anchor.pathname,
+        title: anchor.querySelector<HTMLDivElement>('div.content h4').textContent.trim()
```

Comment:
Wrong type, query is for `HTMLHeading4Element`, but generic is set to `HTMLDivElement`